jujugui/static/gui/src/app/app.js

```diff
@@ -1361,6 +1361,7 @@ YUI.add('juju-gui', function(Y) {
           getUser={this.payment && this.payment.getUser.bind(this.payment)}
           getUserName={getUserName}
           gisf={this.get('gisf')}
+          githubSSHKeys={window.jujugui.sshKeys.githubSSHKeys}
```

Comment:
The name also doesn't indicate what it is. I expected it to be an array or an object, not a function. Maybe `getGitHubSSHKeys` ?